src/components/componentDiff/ComponentDiffTab.js

```diff
@@ -219,6 +221,65 @@ const ComponentVersionDiff = ({
     rightVersion: rightVersion,
   })
 
+  const HandleExportDiff = () => {
+    const addedResources = []
+    const removedResources = []
+
+    diff.components_changed?.forEach(component => {
+
+      const formatResourceName = (resource, component) => {
```

Comment:
If you define this function one level higher, you wouldn't need to pass it to `formatResourceForYaml` anymore.